### PR TITLE
TF Lite added missing axis test cases for ArgMinMax

### PR DESCRIPTION
**Pull request by PariksheetPinjari909**
This PR includes following changes:

1. Adds test case for axis data type
2. Adds missing int8_t data type support for argminmax 
3. Adds error log for better understanding

### CODE REVIEW

File: tensorflow/lite/kernels/arg_min_max_test.cc [hunk 1]
```diff
@@ -152,6 +148,38 @@ TEST(ArgMaxOpTest, GetMaxArgOutput64) {
   EXPECT_THAT(model.GetOutputShape(), ElementsAreArray({1, 1, 2}));
 }
 
+TEST(ArgMaxOpTest, GetMaxArgAxis64) {
```
Comment (renjie-liu) [inline tensorflow/lite/kernels/arg_min_max_test.cc]: nit: Is

File: tensorflow/lite/kernels/arg_min_max_test.cc [hunk 2]
```diff
@@ -207,6 +235,38 @@ TEST(ArgMinOpTest, GetMinArgOutput64) {
   EXPECT_THAT(model.GetOutputShape(), ElementsAreArray({1, 1, 2}));
 }
 
+TEST(ArgMinOpTest, GetMinArgAxis64) {
```
Comment (renjie-liu) [inline tensorflow/lite/kernels/arg_min_max_test.cc]: same here

File: tensorflow/lite/kernels/arg_min_max_test.cc (hunk 1 above)
Comment (PariksheetPinjari909) [reply tensorflow/lite/kernels/arg_min_max_test.cc]: @renjie-liu I didn't get it. Can you elaborate? TIA.

Comment (renjie-liu) [reply tensorflow/lite/kernels/arg_min_max_test.cc]: sorry my bad, miss read it

**Review by renjie-liu — APPROVED** [review]

**Pull request merged**